tasks/channels_anti_entropy_sync.py

```diff
@@ -0,0 +1,67 @@
+import discord
+import os
+import datetime
+from discord.ext import tasks
+from APIs.SweccAPI import SweccAPI
+import dotenv
+
+import logging
+
+logging.basicConfig(
+    level=logging.INFO, format="[%(asctime)s] [%(levelname)s] %(name)s: %(message)s"
+)
+
+dotenv.load_dotenv()
+
+swecc_api = SweccAPI()
+
+
+def get_channel_type(channel):
+    if isinstance(channel, discord.TextChannel):
+        return "TEXT"
+    elif isinstance(channel, discord.VoiceChannel):
+        return "VOICE"
+    elif isinstance(channel, discord.CategoryChannel):
+        return "CATEGORY"
+    elif isinstance(channel, discord.StageChannel):
+        return "STAGE"
+    elif isinstance(channel, discord.ForumChannel):
+        return "FORUM"
+    else:
+        return "UNKNOWN"
```

Comment:
Channels with a type of `UNKNOWN` would be rejected by the backend. Are you sure that `UNKNOWN` is a possible return value of this function? If not, we can just get rid of it and know for sure that `get_channel_type` only returns types supported by the API.

See comment below for an alternative.

Reply:
yeah i can just abort on bot if type unknown